### DISCUSSION TITLE
RLS policy based on user metadata

### DISCUSSION THREAD

**Discussion by saurabhdoteth** [supabase/supabase#13091] [Questions]
my user object has metadata called `role`, I want the row to be updated if `role === 'ADMIN'`, how the policy should look like?

**Comment by GaryAustin1** [comment]
You can use the auth.jwt() function to get the metadata in the jwt.  https://supabase.com/docs/guides/auth/row-level-security#helper-functions

But user metadata is NOT secure.  The user can change it any time with an api call to updateUser()...   Better to use app metadata for that.   https://github.com/supabase-community/supabase-custom-claims
